e2e_test/batch/types/cast.slt.part

```diff
@@ -102,3 +127,28 @@ query T
 values(interval '-50 h'::time);
 ----
 22:00:00
+
+query T
+values('12:34:56'::time::varchar);
+----
+12:34:56
+
+query T
+values(interval '-3 day'::varchar);
+----
+-3 days 00:00:00
+
+query T
+values('2020-01-01'::date::varchar);
+----
+2020-01-01
+
+query T
+values('2020-01-01 12:34:56'::timestamp::varchar);
+----
+2020-01-01 12:34:56
+
+query T
+values('2020-01-01 12:34:56 -08:00'::timestamp with time zone::varchar);
```

Comment:
Will this case fail when it is run in a different timezone? I guess you assume the runner is in UTC+8 (the buildkite server is deployed in Singapore UTC+8

Reply:
No, the current placeholder implementation does not rely on timezone from the running environment. It accepts input with any timezone (e.g. `-08:00` here), and always output in UTC. To make it less confusing, I will update this test case to a less common timezone rather than PST (UTC-8).

In the future, it does relay on `TimeZone` of the session:
* When input does not contain timezone, it will be assumed to be in session TimeZone. We consider this as invalid input as of now.
* As for output, it will be in session TimeZone rather than in UTC.

Reply:
No bother to do that, I think I understand your point.